scripts/ci/pr-bot/.gitignore

```diff
@@ -0,0 +1,41 @@
+lib-cov
+*.seed
+*.log
+*.csv
+*.dat
+*.out
+*.pid
+*.gz
+*.swp
+
+pids
+logs
+results
+tmp
+
+# Build
+public/css/main.css
```

Comment:
Why is this css file ignored? It seems a bit strange that only a single css file is built.

Reply:
You're right, we don't need that - removed